bin/ch/WScriptJsrt.cpp

```diff
@@ -725,7 +725,7 @@ bool WScriptJsrt::InstallObjectsOnObject(JsValueRef object, const char* name, Js
     memcpy(path, msg, (size_t)str_len);          \
     path[str_len] = char(0)
 
-void GetBinaryLocation(char *path, const uint32_t size)
+void GetBinaryLocation(char *path, const unsigned size)
```

Comment:
I'm all for "hope" but I've been paranoid these days about what happens when we make wrong assumptions :P (@digitalinfinity will probably agree)